### PR TITLE
NEWS: more entries and some rewordings

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -196,14 +191,20 @@ CHANGES WITH 243 in spe:
...
-          enabled).
+          default. If this option is set to 'no-negative', negative answers are
+          not cached while the old cache heuristics are used positive answers.
+          The default remains unchanged.
 
         * The predictable naming scheme for network devices now supports
           generating predictable names for "netdevsim" devices.
 
+          Moreover, the "en" prefix was dropped the ID_NET_NAME_ONBOARD udev
+          property.
```
Comment (poettering) [inline NEWS]: something is wrong in this sentence?

File: NEWS [hunk 2]
```diff
@@ -196,14 +191,20 @@ CHANGES WITH 243 in spe:
...
+
+          Those two changes form a new net.naming-policy entry. Distributions
```
Comment (poettering) [inline NEWS]: maybe suffix the net.naming-policy with "=" as we usually do for variables?

**Comment by poettering** [comment]
might also need a new "ninja -C build git-contrib" update?

**Comment by keszybz** [comment]
Updated.

> might also need a new "ninja -C build git-contrib" update?

It seems not needed.

**Pull request merged**